### DISCUSSION TITLE
No hypno deafen setting and Nicknames in magic

### DISCUSSION THREAD

**Discussion by EmilyBondage** [littlesera/LSCG#360] [Ideas]
Im suggesting a few features
 - a setting so when hypnoed you aren't deafened so you can get orders from anyone
 - a spell part or extension to polymorph or outfit to change the persons nickname, with a setting like pronouns to block
 - a setting in polymorph and outfit to not block the wardrobe

**Comment by littlesera** [comment]
FYI, when hypnotized you will be able to read the chat of anyone that says your name in the same line, otherwise just the person who hypnotizes you.
